app/models/features.rb

```diff
@@ -1,7 +1,7 @@
 # Co-erces an HStore object of feature flags into a ruby model
 class Features
   include Virtus.model
-  attribute :check_in, Boolean, default: false
+  attribute :check_in, Boolean, default: true
```

Comment:
Please don't change the default of this. It's off so that we can release from primary to production without worrying about exposing a half-finished feature.